Use deps from crates instead of bundled in the repo.

--- ripgrep-0.8.1/Cargo.toml
+++ ripgrep-0.8.1/Cargo.toml
@@ -29,16 +29,13 @@
 name = "integration"
 path = "tests/tests.rs"
 
-[workspace]
-members = ["grep", "globset", "ignore", "termcolor", "wincolor"]
-
 [dependencies]
 atty = "0.2.2"
 bytecount = "0.3.1"
 encoding_rs = "0.7"
-globset = { version = "0.3.0", path = "globset" }
-grep = { version = "0.1.8", path = "grep" }
-ignore = { version = "0.4.0", path = "ignore" }
+globset = "0.3.0"
+grep = "0.1.8"
+ignore = "0.4.0"
 lazy_static = "1"
 libc = "0.2"
 log = "0.4"
@@ -47,7 +44,7 @@
 num_cpus = "1"
 regex = "0.2.4"
 same-file = "1"
-termcolor = { version = "0.3.4", path = "termcolor" }
+termcolor = "0.3.4"
 
 [dependencies.clap]
 version = "2.29.4"
